src/main/java/qamasters/pages/HomePage.java

```diff
@@ -0,0 +1,4 @@
+package qamasters.pages;
+
+public class HomePage {
```

Comment:
What is the purpose of this class, there is no code in this

Reply:
just created for the actions on home page, I will add the code in testcase-2

Reply:
ok